shared/js/background/redirect.es6.js

```diff
@@ -109,6 +107,11 @@ function handleRequest (requestData) {
                     }
                 }
 
+                if (!window.safari) {
```

Comment:
We initially released element hiding to only chrome + firefox for a couple reasons (safari extensions are reaching end of life and this content script will require significant changes in order to work similarly in safari since safari's messaging is a bit different). 

Previously the element hiding process was kicked off by the injected content script messaging the background page, which only happened in chrome + firefox. Now that the background page is kicking off the process, I added this flag here to keep things roughly the same.

I'll verify with the team which platforms we need to support here and get back with an answer.